pkg/models/model_plan.go

```diff
@@ -5,9 +5,10 @@ import "github.com/google/uuid"
 // ModelPlan is the top-level object for an entire draft model plan
 type ModelPlan struct {
 	baseStruct
-	ModelName string      `json:"modelName" db:"model_name"`
-	Archived  bool        `json:"archived" db:"archived"`
-	Status    ModelStatus `json:"status" db:"status"`
+	ModelName    string      `json:"modelName" db:"model_name"`
+	Abbreviation *string     `json:"abbreviation" db:"abbreviation"`
```

Comment:
Since you changed this to be just abbreviation, all the SQL will need to be updated as well, or the db tag needs to be `model_abbreviation`. I believe this is why db seed and the tests are failing.

Reply:
I think it's probably best practice to just use the same field name in all spots (eg update db to just use abbreviation if that's what you want to use on the model)